scripts/keri/cf/verifier-config-public.json

```diff
@@ -29,5 +29,9 @@
     "https://gleif-it.github.io/oobi/EH6ekLjSr8V32WyFbGe1zXjTzFs9PkTYmupJ9H65O14g",
     "https://gleif-it.github.io/oobi/EBfdlu8R27Fbx-ehrqwImnK-8Cm79sqbAQ4MmvEAYqao"
   ],
-  "LEIs": []
+  "LEIs": [],
+
+  "acceptedRoles": [
+    "EBA_DATA_ADMIN"
```

Comment:
```suggestion
    "EBA Data Submitter"
```

Reply:
It was good to test EBA_DATA_ADMIN as an override, but the default configuration should be EBA Data Submitter